openfeature-provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.java

```diff
@@ -32,58 +30,26 @@ public class ConfidenceFeatureProvider implements FeatureProvider {
   /**
    * ConfidenceFeatureProvider constructor
    *
-   * @param confidence an instance of the Confidence
+   * @param confidenceWrapper obtained via {@link #createConfidenceWrapper(Confidence.Builder)}
    */
-  public ConfidenceFeatureProvider(Confidence confidence) {
-    this.confidence = confidence;
+  public ConfidenceFeatureProvider(ConfidenceWrapper confidenceWrapper) {
+    this.confidence = confidenceWrapper.confidence;
   }
 
-  private static final Logger log =
-      org.slf4j.LoggerFactory.getLogger(ConfidenceFeatureProvider.class);
-
-  /**
-   * ConfidenceFeatureProvider constructor
-   *
-   * @param clientSecret generated from the Confidence portal
-   * @param managedChannel gRPC channel
-   * @deprecated This constructor is deprecated. Please use {@link
-   *     #ConfidenceFeatureProvider(Confidence)} instead.
-   */
-  @Deprecated()
-  public ConfidenceFeatureProvider(String clientSecret, ManagedChannel managedChannel) {
```

Comment:
Removing these public constructors is a breaking change (maybe reconsider them?). 
We should at least rename the PR (and the resulting commit on main) with a `!` to signal this.
`refactor: Add ConfidenceWrapper for OF Metadata injection` -> `refactor!: Add ConfidenceWrapper for OF Metadata injection `

Reply:
On second thought... We've had this marked as deprecated for some versions I think so maybe removing is fine?